client/src/App.js

```diff
@@ -42,7 +44,8 @@ function App() {
         </Switch>
       </main>
       <footer>
-        {location.pathname !== `http://localhost:3000/estate` && <Footer/>}
+      <Footer/>
```

Comment:
You should just add another `Switch` in here, to render the footer only if the path is what you want:

```js
<Switch>
     <Route path={["/path/with/footer", /path/also/with/footer]} />
         <Footer />
     </Route>
</Switch>
```
Also, please never use `localhost:3000/...`. When the app is deployed, the path won't be `localhost...` but something else.